classes/journal/enums/JournalContentOption.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file classes/journal/enums/JournalContentOption.php
+ *
+ * Copyright (c) 2014-2025 Simon Fraser University
+ * Copyright (c) 2003-2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class JournalContentOption
+ *
+ * @brief Describes the content options for a journal.
+ */
+
+namespace APP\journal\enums;
+
+use APP\core\Application;
+use APP\facades\Repo;
+use PKP\context\Context;
+
+enum JournalContentOption: int
+{
+    case ISSUE_TOC = 1;
+    case RECENT_PUBLISHED = 2;
+    case CATEGORY_LISTING = 3;
+
+    /**
+     * Get the default content options for a journal.
+     */
+    public static function default(?Context $context = null): array
+    {
+        $context ??= Application::get()->getRequest()->getContext();
+        
+        if (!$context) {
+            return [static::ISSUE_TOC];
+        }
+
+        return static::getIssueExists($context) ? [static::ISSUE_TOC] : [static::RECENT_PUBLISHED];
+    }
+
+    /**
+     * Get the content options for a journal.
+     */
+    public static function getOptions(?Context $context = null): array
+    {
+        $context ??= Application::get()->getRequest()->getContext();
+        $options = static::getDefaultOptions();
+
+        if (!$context) {
+            return $options;
+        }
+
+        if (!static::getIssueExists($context)) {
+            array_shift($options);
```

Comment:
I think this behaviour should be removed and the issue option should always be available. Consider a newly installed OJS, where there won't be any issues yet; the editor won't be able to choose issue-based publication until after they've published an issue, which will be confusing.